src/components/Dropdown/index.tsx

```diff
@@ -38,7 +38,6 @@ const Dropdown: React.FC<DropdownProps> = ({
         ref={menuButtonRef}
         onClick={() => setIsOpen(!isOpen)}
         data-cy={dataCy}
-        id="searchable-dropdown"
```

Comment:
Any idea why/if this was causing the flicker?

Reply:
Yeah haha it's because the `Dropdown` is used in multiple places. For example, the Status Selector (the dropdown that Sophie could keep opening and closing through the bug) also uses the `Dropdown` which meant it had `id = searchable-dropdown`, so it was connected to the wrong label. 😅 I added an `id` field onto the `Dropdown` just now, lmk what you think!